db/command.go

```diff
@@ -249,3 +251,39 @@ func ApplyFlags(opts *mopt.FindOneOptions, flags int) {
 		opts.SetOplogReplay(true)
 	}
 }
+
+// RunApplyOpsCreateIndex will create index using applyOps.
+// For versions that support collection UUIDs (<3.6) it uses an insert to system indexes.
+// Later versions use the createIndexes command.
+func (sp *SessionProvider) RunApplyOpsCreateIndex(C, DB string, index bson.D, UUID primitive.Binary, supportsUUID bool, result *interface{}) error {
```

Comment:
In the mongomirror code, `UUID` is `*primitive.Binary` and also plays the role of `supportsUUID`.  What's your thinking about changing that?

Reply:
I (mildly) worry about the disconnect between a boolean for "supports" and a UUID that has to be a zero value when supports is false.

Reply:
Yeah good point, that is a bit messy.